doc/modules/classes.rst

```diff
@@ -120,7 +120,6 @@ Functions
    cluster.cluster_optics_dbscan
    cluster.cluster_optics_xi
    cluster.compute_optics_graph
-   cluster.dbscan
```

Comment:
Please revert these changes. They were issues in master that have been patched there.

Reply:
Done.